rosetta/failure/invalid_authorizers.go

```diff
@@ -0,0 +1,29 @@
+// Copyright 2021 Optakt Labs OÜ
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+package failure
+
+import (
+	"fmt"
+)
+
+type InvalidAuthorizers struct {
+	Description Description
+	Have        uint
+	Want        uint
+}
+
+func (i InvalidAuthorizers) Error() string {
+	return fmt.Sprintf("invalid authorizers (have: %d, want: %d): %s", i.Have, i.Want, i.Description)
```

Comment:
At least the error description should include `invalid number of authorizers`. Or is that an index, not a count?

Reply:
No, that's a count of authorizers. Will rephrase the description